flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -429,12 +429,61 @@ def set_config_spw(self, spw_name: str):
     def turn_off_bnd_press_correction(self):
         self.sf_model.set_config("pavbnd", -9999)
 
+    def add_obs_points(self):
+        """add observation points provided in the site toml to SFINCS model"""
+
+        if self.site.attrs.obs_point is not None:
```

Comment:
Can't you use the get_obs_points in the dbs_controller for both of these (add_obs_points and read_zs_points)?

Reply:
But how does the SFINCS adapter know about the database?

Reply:
I'll make a separate issue out of it